metadata-models/src/main/pegasus/com/linkedin/metadata/key/MLFeatureKey.pdl

```diff
@@ -11,10 +11,20 @@ record MLFeatureKey {
   /**
    * Namespace for the feature
    */
-   featureNamespace: string
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "addToFilters": true
```

Comment:
So we want filters for all the namespaces? Number of namespaces might be a lot right?

Reply:
Hmm I suppose not – will remove